blockchain/state_migration.go

```diff
@@ -528,3 +531,141 @@ func (bc *BlockChain) StopWarmUp() error {
 
 	return nil
 }
+
+// StartCollectingTrieStats collects state or storage trie statistics.
+func (bc *BlockChain) StartCollectingTrieStats(contractAddr common.Address) error {
+	block := bc.GetBlockByNumber(bc.lastCommittedBlock)
+	if block == nil {
+		return fmt.Errorf("Block #%d not found", bc.lastCommittedBlock)
+	}
+
+	mainTrieDB := bc.StateCache().TrieDB()
+	cache := mainTrieDB.TrieNodeCache()
+	if cache == nil {
+		return fmt.Errorf("target cache is nil")
+	}
+	db := state.NewDatabaseWithExistingCache(bc.db, cache)
+
+	startNode := block.Root()
+	// If the contractAddr is given, start collecting stats from the root of storage trie
+	if !common.EmptyAddress(contractAddr) {
```

Comment:
There's `isEmptyAddress` function in `reward` package. I think those two functions do the same thing, so removing `isEmptyAddress` would be better.. Can you add `//TODO` mark at `isEmptyAddress`?

Reply:
@yoomee1313 I've opened a PR https://github.com/klaytn/klaytn/pull/741  to use `common.EmptyAddress` instead of `reward.isEmptyAddress`. #741 will be merged after this PR. Thanks!